### DISCUSSION TITLE
Cannot merge PR with new workflow

### DISCUSSION THREAD

**Discussion by parkr** [community/community#38590] [Mobile] [answered]
Hey GitHub,

I received a number of PRs from lgtm-com bot to migrate my CodeQL integrations to Actions. However, I get an error when I try to merge the PRs on mobile:

![image](https://user-images.githubusercontent.com/237985/200635083-b7d151fd-dbf6-4398-8d58-f2011c8ee769.png)

It appears the mobile app doesn't have the `Workflow` scope for its oauth app. Could that be updated so that we can merge/modify workflows within GitHub for Mobile?

Here's the PR:

![image](https://user-images.githubusercontent.com/237985/200635302-add566f7-a89e-45ea-8fd7-09ff70b2c8f5.png)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.89.0 (83472182)
Device: iPhone 13 Pro (iOS 16.1)
</details>

**Comment by david-engelmann** [comment]
Have you enabled actions for the repo?

**Reply by parkr** [reply]
Yes, Actions is enabled. https://github.com/parkr/just-me/tree/main/.github/workflows

I believe the issue is with the oauth scopes granted to GitHub Mobile. When building an app you can select the scopes you need and users who grant that app access to their account provide it that scoped access.

I'll see if logging out and logging back in fixes the issue with scopes. It could be that I logged in (years ago) before the workflow scope was added.

**Reply by parkr** [reply]
Logging out and back in does not fix the issue. I can merge from GitHub.com on mobile so I'll do that for now.

**Comment by aconsuegra** [comment]
Hi @parkr, thanks for getting in touch! The Mobile apps don't request the `workflow` scope, so it is expected that you can't merge that pull request. Without that scope, it is possible to merge pull requests that add workflows if they are created by users, but the scope is required in case a pull request is created by an app.

We will discuss internally about adding the scope to the token. Thanks again!

**Reply by parkr** [reply]
Thanks, Antonio! Hope you are doing well :)

**Comment by spaltrowitz** [comment, marked as answer]
Hi @parkr & @david-engelmann, thanks for your feedback. A feature related to your feedback has now shipped! 🚢 https://github.blog/changelog/2023-05-11-edit-workflow-files-on-github-mobile/ cc @aconsuegra
